lib/params.js

```diff
@@ -21,9 +21,9 @@ function hasScssInWorkingDir (wd) {
 function getIndentationFromEditorConfig (from, hasScss) {
   var config
   if (hasScss) {
-    config = editorconfig.parseSync(path.resolve(process.cwd(), '*.scss'))
+    config = editorconfig.parseSync(path.resolve(from, '*.scss'))
```

Comment:
`getIndentationFromEditorConfig()` is only call by `params()`, `process.cwd()` will be pass.